README.md

```diff
@@ -7,6 +7,8 @@ Docker image for Swift all-in-one demo deployment
 
 This is an attempt to dockerize the instructions for a [Swift All-in-one deployment](https://docs.openstack.org/swift/latest/development_saio.html).
 
+[Source code is available here](https://github.com/NVIDIA/docker-swift)
```

Comment:
seems a little self referential, i probably don't understand where we'd distribute this file that's not already pointing to this repo.  Maybe https://hub.docker.com/r/dockerswiftaio/docker-swift

Do we want to publish a 2.28.0 tag or whatever?

Reply:
yes, i always find the dockerhub page first and then have to figure out how to get back to the repo

i don't particularly care about a 2.28.0 tag. do you?

Reply:
i legit forgot about this repo!  so i don't know what it's used for or if anyone that uses it would want a 2.28.0 tag.  If you're the only user and and don't care about the tag - then no, I don't care either